backend/cron.js

```diff
@@ -0,0 +1,92 @@
+const PullRequest = require('./services/PullRequest')
```

Comment:
What do you think of renaming the script to `backend/worker.js` or even `backend/workers/fetch-pull-requests.js`?

Reply:
done!